From e276d95a5d611bb0dd13984d6e1ac72c2145be62 Mon Sep 17 00:00:00 2001
From: Calvin Johnson <calvin.johnson@nxp.com>
Date: Mon, 8 Jul 2019 15:48:38 +0530
Subject: [PATCH] configs: ls1043avnprdb: enable FDT_SEQ_MACADDR_FROM_ENV

Enable FDT_SEQ_MACADDR_FROM_ENV to fetch MAC address
sequentially from environment variables

Signed-off-by: Calvin Johnson <calvin.johnson@nxp.com>
---
 include/configs/ls1043avnprdb.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/ls1043avnprdb.h b/include/configs/ls1043avnprdb.h
index 81908e5835..9e6175a3fa 100755
--- a/include/configs/ls1043avnprdb.h
+++ b/include/configs/ls1043avnprdb.h
@@ -302,6 +302,7 @@ board/freescale/ls1043ardb/ls1043avnprdb_rcw_emmc_nand.cfg
 #define CONFIG_PHY_MICREL_KSZ9031
 #endif
 
+#define FDT_SEQ_MACADDR_FROM_ENV
 #ifdef CONFIG_SYS_DPAA_FMAN
 #define CONFIG_FMAN_ENET
 
-- 
2.17.1

